README.md

```diff
@@ -26,6 +26,18 @@ suggested by the README.
 The addon files will be placed in `../../xbmc/kodi-build/addons` so if you build Kodi from source and run it directly 
 the addon will be available as a system addon.
 
+### Linux for regular users
```

Comment:
What is a "regular user", compared to a developer? I would not make assumptions who is using this, but write for what it can be used.

Reply:
Good point. How would you like me to rephrase this?

Reply:
Make a suggestion, please.

Reply:
How about 'for system wide installations, also suitable for packaging'?

Reply:
However, I don't have a clue as to what the original instructions should be called to distinguish them in the right way from the new ones...